style.css

```diff
@@ -0,0 +1,44 @@
+#main
+    {
+      background-color:whitesmoke;
+    }
+
+#title
+    {
+       padding-left: 600px;
+       font-size: 40px;
+       font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+    }
+    p{
+        padding-left: 575px;
+        font-size: 30px;
+        font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+    }
```

Comment:
**Required Changes ♻️**

Check the comments under the review.

- [ ]  Why haven't you use Linters, I think it will format your CSS file perfectly.

Reply:
- [ ] [REQUIRED] Kindly note to always include an explanation, a "why", for each requested change. Why should Linters be used?
For example, "Please ensure you use Linters as this helps adhere to code standards"
- [ ] Please note that your comments should be phrased politely as suggestions. For example, This is a polite suggestion: "I think it would be better to use Linters", "Kindly ensure to use Linters"